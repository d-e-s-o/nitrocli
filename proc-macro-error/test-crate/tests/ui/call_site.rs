extern crate test_crate;
use test_crate::make_fn;

make_fn!(abort_call_site);

fn main() {}
