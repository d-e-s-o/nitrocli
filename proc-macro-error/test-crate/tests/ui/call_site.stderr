error: abort_call_site! 2+ args test

  = help: help message test

 --> $DIR/call_site.rs:4:1
  |
4 | make_fn!(abort_call_site);
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^ in this macro invocation
